tests/test_multidict.py

```diff
@@ -387,6 +387,17 @@ def test_and2(self, cls: Type[MutableMultiMapping[str]]) -> None:
 
         assert {"key"} == {"key", "key2"} & d.keys()
 
+    def test_and_not_implemented(self, cls: Type[MutableMultiMapping[str]]) -> None:
+        d = cls([("key", "value1")])
+
+        with pytest.raises(TypeError):
+            operator.and_(d.keys(), 1)
```

Comment:
How about just
```suggestion
            d.keys() and 1
```
? I think it'd be clearer to an inexperienced reader.